README.md

```diff
@@ -1,6 +1,8 @@
 # AWS Support Cases Aggregator
 _AWS Support Cases aggregation for a multi-account organization_
 
+[![ci](https://github.com/sSnapchat/aws-support-tickets-aggregator/ci/badge.svg?branch=master)](https://github.com/Snapchat/aws-support-tickets-aggregator/actions?query=workflow%3Aci+branch%3Amaster)
```

Comment:
I think the badge link here isn't correct. Should be Snapchat, not sSnapchat.

Reply:
corrected